CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add items schema for `.kubectlImage`.
```

Comment:
The word "items" suggests that this is an array property.

```suggestion
- Add schema for `.kubectlImage`.
```

Reply:
I just reused that of your previous change 😅